src/fraud_credit_cards/fraud_model.py

```
@@ -0,0 +1,37 @@
+from sklearn.pipeline import Pipeline
+from catboost import CatBoostClassifier
+from sklearn.metrics import accuracy_score, classification_report, precision_score, recall_score, f1_score, roc_auc_score, mean_squared_error, r2_score
```

Ruff (0.8.2)

`sklearn.metrics.classification_report` imported but unused

Remove unused import

(F401)